openslides_backend/shared/interfaces.py

```diff
@@ -79,6 +79,8 @@ class WSGIApplication(Protocol):  # pragma: no cover
     Interface for main WSGI application class.
     """
 
+    services: Any
```

Comment:
Is this line necessary? I do no see which change in some other file would cause mypy fail.

Reply:
Mypy complained when you had an object of type `WSGIApplication` and tried to access the services, I think I had to cast to `Any` every time.